cypress/utils/table.js

```diff
@@ -192,7 +192,7 @@ export const testSorting = (columns, isTableExpandable = false) => {
 export const testFilters = (filters) => {
   filters.forEach((filter, index) => {
     it(`filters by ${filter.urlParam}`, () => {
-      cy.get('button[aria-label="Conditional filter"]').click();
+      cy.get('button[aria-label="Conditional filter toggle"]').click();
```

Comment:
This change was necessary after updating `@redhat-cloud-services/frontend-components`